interop/xds/custom_lb_test.go

```diff
@@ -66,7 +66,7 @@ func (s) TestCustomLB(t *testing.T) {
 			}
 			rpcBMD := md.Get("rpc-behavior")
 			if len(rpcBMD) != 1 {
-				errCh.Send(errors.New("only one value received for metadata key rpc-behavior"))
+				errCh.Send(errors.New("only one value should be received for metadata key rpc-behavior"))
```

Comment:
How about also including the actual number of values received in the error message?

`errCh.Send(errors.New("received %d values for metadata key rpc-behavior, want 1"))`